.github/workflows/release-build-outputs.yml

```diff
@@ -0,0 +1,108 @@
+# HMS Networks Solution Center
```

Comment:
Not required, but there is a new version of release-build-outputs on the starter project repository now. It might be good to synchronize this with the latest, which also includes the .JAR and jvmrun files directly in the github release.